static/js/components/NewShift.jsx

```diff
@@ -27,12 +27,37 @@ const NewShift = () => {
   }
 
   const handleSubmit = async ({ formData }) => {
-    formData.start_date = formData.start_date.replace("+00:00", "");
-    formData.end_date = formData.end_date.replace("+00:00", "");
-    const result = await dispatch(submitShift(formData));
-    if (result.status === "success") {
-      dispatch(showNotification("Shift saved"));
-      dispatch(fetchShifts());
+    if (!formData.multipleDays) {
+      formData.start_date = formData.start_date.replace("+00:00", "");
+      formData.end_date = formData.end_date.replace("+00:00", "");
+      delete formData.multipleDays;
+      const result = await dispatch(submitShift(formData));
+      if (result.status === "success") {
+        dispatch(showNotification("Shift saved"));
+        dispatch(fetchShifts());
+      }
```

Comment:
Shouldn't there be an else with an error message? or is that sent up by the `self.error`?

Reply:
So, as we are using liveValidate in the form, no error should occur as the user cant even submit the shift if the input is wrong. For the feature where you create a shift directly in the calendar, without having to specify an else, it already displays an error message (lets say you forgot the name, it will print (missing required parameter 'name')